### PR TITLE
py-connectome-manipulator: Release 0.0.10

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

**Review by matz-e — CHANGES_REQUESTED** [review]
The version constraints on some of the dependencies can now be dropped; `py-submitit` altogether, too!

**Comment by eleftherioszisis** [comment]
@matz-e better?

File: bluebrain/repo-bluebrain/packages/py-connectome-manipulator/package.py [hunk 1]
```diff
@@ -27,17 +27,14 @@ class PyConnectomeManipulator(PythonPackage):
...
     depends_on("py-pandas", type=("build", "run"))
     depends_on("py-progressbar", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
     depends_on("py-scikit-learn", type=("build", "run"))
     depends_on("py-voxcell", type=("build", "run"))
-    depends_on("py-pyarrow+parquet+dataset@3.0.0:", type=("build", "run"))
-    depends_on("py-jsonpickle", type=("build", "run"))
+    depends_on("py-pyarrow+parquet+dataset", type=("build", "run"))
     depends_on("py-distributed", type=("build", "run"), when="@0.0.6:")
     depends_on("py-dask-mpi", type=("build", "run"), when="@0.0.6:")
```
Comment (matz-e) [inline bluebrain/repo-bluebrain/packages/py-connectome-manipulator/package.py]:
```suggestion
    depends_on("py-distributed", type=("build", "run"))
    depends_on("py-dask-mpi", type=("build", "run"))
```

**Review by matz-e — APPROVED** [review]

**Comment by eleftherioszisis** [comment]
Not sure why matplotlib is not found given that it is a dependency in py-neurom.

**Comment by matz-e** [comment]
Purged PR directory, rebuilding from scratch.

The spec JSON files on disk had the right dependencies set up. Not sure why that one module did not contain matplotlib. Let's see.

**Pull request merged**